src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.Android.cs

```diff
@@ -343,17 +344,14 @@ protected override Size ArrangeOverride(Size finalSize)
 				else
 				{
 					// The layout is different and needs to be rebuilt.
-					UpdateLayout(ref _arrangeLayout, finalSize, exactWidth: true);
+					UpdateLayout(ref _arrangeLayout, arrangeSize, exactWidth: true);
 				}
 
 				if (originalArrangeLayout != _arrangeLayout)
 				{
 					UpdateNativeTextBlockLayout();
 				}
 
```

Comment:
did you forget to add back the Padding?

Reply:
The control will return `finalSize`, so no need to add back the padding.

That's how it was done before, I didn't change this logic.

Reply:
![image](https://user-images.githubusercontent.com/8270914/67959193-b8933280-fbce-11e9-9592-22af4bc7c56e.png)
?